cropharvest/eo/eo.py

```diff
@@ -55,8 +57,23 @@ def __init__(self, data_folder: Path = DATAFOLDER_PATH) -> None:
         # allows for easy checkpointing
         self.cur_output_folder = f"{self.output_folder_name}_{str(date.today()).replace('-', '')}"
 
-        self.labels = geopandas.read_file(data_folder / LABELS_FILENAME)
-        self.labels["export_end_date"] = pd.to_datetime(self.labels["export_end_date"])
+        self.labels = self.default_labels if labels is None else labels
+        self.using_default = labels is None
+        for expected_column in ["start_date", "end_date", "lat", "lon"]:
```

Comment:
Why not use the Columns class here?

Reply:
`start_date` & `end_date` are not columns in the default labels - but I can replace the lat and lon column names with whats in the class.

Reply:
Yeah let's replace lat, lon for sure. Would it make sense to store the start and end date in the default labels?

Reply:
Updated